src/get-started/light-client.md

```diff
@@ -316,34 +439,29 @@ if (typeof wallet === 'undefined') {
 }
 ```
 
-Basically, the proof from the Timestamping Demo comprises the same validation
+Basically, the proof from the Timestamping Service comprises the same validation
 levels as described above. Specifically, to obtain confirmation for the
 timestamp data, the proof validates the block, the root table and the table of
 timestamps. The data on the timestamp can then be extracted from the validated
 timestamps table.
 
-Meanwhile, the proof we presently investigate contains another level that refers
-to the validation of transactions in respect of each wallet.  
+Meanwhile, the proof we presently investigate contains another level. This level
+refers to the validation of transactions in respect of each wallet (wallet
+history).  
 
 First, we obtain a proof for all transactions in the wallet. It will contain
-transaction hashes as well as statuses thereof. In our example we obtain
-information for the whole wallet transaction history, however, any suitable
-range may be selected:
+transactions hashes as well as statuses thereof. In our example we obtain
+information for the whole wallet transactions history. However, any suitable
+range of the history may be selected:
 
 ```javascript
-const TransactionMetaData = Exonum.newType({
-  fields: [
-    { name: 'tx_hash', type: Exonum.Hash },
-    { name: 'execution_status', type: Exonum.Bool }
-  ]
-})
 
 const transactionsMetaData = Exonum.merkleProof(
-  wallet.history_hash,
+  Exonum.uint8ArrayToHexadecimal(new Uint8Array(wallet.history_hash.data)),
```

Comment:
Nit / off-topic: Isn't it possible to use `Buffer` utils for hex <-> Uint8Array conversion? IMO, this kind of reuse would be preferable to rolling out our own utils.

Reply:
Thanks, looks very interesting. I will investigate if it is possible to do such converting and if so I will implement it into Light client, demos and tutorial.

Reply:
The only reservation I can think of is that `Buffer` is not readily available in browser environments, but it can be easily added there by using [the `buffer` package](https://www.npmjs.com/package/buffer), which provides fully Node-compatible interface.